### DISCUSSION TITLE
how do I change the playground endpoint type from stream_log to invoke

### DISCUSSION THREAD

**Discussion by SoulEvill** [langchain-ai/langserve#547] [Q&A] [answered]
I created a sequence of runnable and would like to test it out in the playground. the endpoint is up but when I click on start - maybe because of complicated chain that I have it was returning repeated result. I looked in the server log and realized the playground 'start' button was calling the stream_log - i was wondering if this behavior could be config - I would like to change it from stream_log to invoke.

sample log information from server when click on start button from playground: 
"127.0.0.1:49397 - "POST /xxxx/stream_log HTTP/1.1" 404 Not Found"

any help would be really appreciated! Thanks!

**Comment by eyurtsev** [comment, marked as answer]
That's not possible currently. The playground uses the astream_log API. We might make it more configurable in the future.

**Comment by SoulEvill** [comment]
Thanks! I will turn off the playground for now and thanks for confirmation
